bin/describe/ls_like_cmd.mli

```diff
@@ -0,0 +1,8 @@
+open Import
+
+val term :
+     (   Dune_rules.Main.build_system
+      -> Path.Source.t
+      -> Path.t
+      -> string list Action_builder.t)
```

Comment:
These arguments need some labeling and/or documentation.

1. What's the Path.Source.t?
2. What's the Path.t?
3. What's the list of strings being returned?

Reply:
They are all hidden details now.